services/device-registry/src/main/fabric8/deployment.yml

```diff
@@ -23,6 +23,20 @@ spec:
         - sh
         - -c
         - 'cp -u /tmp/hono/example-credentials.json /var/lib/hono/device-registry/credentials.json'
+      # we use another init container to populate the registry
```

Comment:
I can't see why we should

Reply:
I know, as I commented already earlier. This solution is working, but I will dig deeper into why the combination with the already existing init container for some reasons did not work.
I am sure there will be a solution, but will not put it top prio (but will do that soon!).

Reply:
does that mean that you can't do it in this PR?

Reply:
Wanted to keep this PR focussed on tests only.

There is the implementation PR #468 for the service itself, which contained this change. So I would rather push it on top there to keep the PRs as clean as possible.

(The change was contained in the squashed commit from the implementation PR).